control-userlayer/src/main/java/org/oskari/control/userlayer/CreateUserLayerHandler.java

```diff
@@ -235,115 +264,9 @@ public void handleAction(ActionParameters params) throws ActionException {
                 JSONHelper.putValue(userLayer, "warning", featuresSkipped);
             }
             response.getWriter().print(userLayer);
```

Comment:
You could just do ResponseHelper.writeResponse(params, userLayer)?

Reply:
Yea didn't want to touch this beast, no idea if the IE iframe workaround thingy is still valid (seems that it requires "Content-Type: text/plain").

Reply:
The frontend doesn't use iframe for the upload anymore so the hack shouldn't be needed anymore.